File: engine/base/src/main/java/client/database/data/BbsThreadReplyData.java
package client.database.data;

public class BbsThreadReplyData {
   private int threadId;

   private int replyId;

   private int posterCharacterId;

   private long timestamp;

   private String content;

   public BbsThreadReplyData(int replyId, int posterCharacterId, long timestamp, String content, int threadId) {
      this.replyId = replyId;
      this.posterCharacterId = posterCharacterId;
      this.timestamp = timestamp;
      this.content = content;
      this.threadId = threadId;
   }

   public int getThreadId() {
      return threadId;
   }

   public int getReplyId() {
      return replyId;
   }

   public int getPosterCharacterId() {
      return posterCharacterId;
   }

   public long getTimestamp() {
      return timestamp;
   }

   public String getContent() {
      return content;
   }
}

File: engine/model/src/main/scala/client/database/data/BbsThreadReplyData.scala
package client.database.data

class BbsThreadReplyData(private var _threadId: Int, private var _replyId: Int, private var _posterCharacterId: Int,
                         private var _timestamp: Long, private var _content: String) {
  def threadId: Int = _threadId

  def replyId: Int = _replyId

  def posterCharacterId: Int = _posterCharacterId

  def timestamp: Long = _timestamp

  def content: String = _content
}
